src/components/Container/index.tsx

```diff
@@ -3,24 +3,28 @@ import styled from 'styled-components'
 
 export interface ContainerProps extends LayoutProps {
   fluid?: boolean
+  inner?: boolean
+  noPadding?: boolean
 }
 
 export const Container = styled.div<ContainerProps>(
   props => ({
     width: '100%',
-    paddingLeft: props.theme.space[3],
-    paddingRight: props.theme.space[3],
+    paddingLeft: props.noPadding ? 0 : props.theme.space[3],
+    paddingRight: props.noPadding ? 0 : props.theme.space[3],
```

Comment:
I think we should keep `Container` as it is; minimal without many properties. What do you think?

Reply:
Well... that's why I added a noPadding prop with default value.
So on other components or pages, the container will work as it is. If we add "noPadding: true" then it will remove the padding.

Reply:
Could you please give me an idea how I can improve this?
Remove the noPadding prop or should I create another component without padding?